src/components/common/Questionnaire/ExpandedModal/ExpandedModal.module.css

```
@@ -121,6 +126,12 @@
   background-color: #f9f9f9;
 }
 
+.enabled {
```

Optional :-)

```suggestion
.nextStepButton:not(:disabled) {
```

如此一來就不需要 `styles.enabled`

---

對耶，由 `:not(:disabled)` 來控制就好，這樣也不需要 `cn` 連接 `styles.nextStepButton`。覺得很棒，感謝！